backends/dart_minerva/lib/builders/endpoints_builder.dart

```diff
@@ -0,0 +1,40 @@
+import 'dart:convert';
+
+import 'package:minerva/minerva.dart';
+
+class EndpointsBuilder extends MinervaEndpointsBuilder {
+  @override
+  void build(Endpoints endpoints) {
+    endpoints.get('/', _root);
+
+    endpoints.post('/echo', _echo);
+
+    endpoints.post('/file_upload', _fileUpload);
+
+    endpoints.post('/json_obj', _jsonObject);
+  }
+
+  dynamic _root(ServerContext context, MinervaRequest request) {
+    return {'response': 'Hello, world!'};
+  }
+
+  dynamic _echo(ServerContext context, MinervaRequest request) async {
+    var json = await request.body.asJson();
+
+    return {"response": "Hello, ${json['name']}!"};
+  }
+
+  dynamic _fileUpload(ServerContext context, MinervaRequest request) async {
+    var form = await request.body.asForm();
+
+    var field = form.data['benchmark'] as FormDataFile;
+
+    return utf8.decode(field.bytes).length;
```

Comment:
Is this working? because other frameworks didn't allow me to return `int` type

Reply:
It will convert int to String when it sends the response.

If you return any value (without using a separate Result class, which can be used to configure the return code and so on), then the framework converts this to String using the toString method. The exception is Map<String, dynamic>, if you return an object of this type, the response will contain JSON.

Reply:
Also, the standard status code in this case will be the code 200.

Reply:
Of course, this behavior may seem unclear. And this is described in the documentation.

Reply:
Oh, that's nice.